docs/openj9_signals.md

```diff
@@ -59,17 +59,19 @@ Note that certain signals on VM threads cause OpenJ9 to shutdown. An application
 | `SIGINT (2)`      | Int  | Interactive attention (CTRL-C), VM exits normally                  | `-Xrs` |
 | `SIGTERM (15)`    | Int  | Termination request, VM exits normally                             | `-Xrs`  |
 | `SIGHUP (1)`      | Int  | Hang up, VM exits normally                                         | `-Xrs` |
+| `SIGUSR2 (12)`    | Int  | User-defined signal for triggering a dump agent                    | `-Xrs` |
 | `SIGQUIT (3)`     | Con  | Quit signal from a terminal, which triggers a Java dump by default | `-Xrs` |
 | `SIGTRAP (5)`     | Con  | Used by the JIT                                                    | `-Xrs` or `-Xrs:sync` |
 | `SIGRTMIN (34)`   | Con  | Used by the VM for thread introspection                       | - |
-| `SIGRTMIN +1 (35)`| Con  | Used by the VM for Runtime Instrumentation (Linux for IBM Z systems only)| - |
+| `SIGRTMIN +1 (35)`| Con  | Used by the VM for Runtime Instrumentation (Linux for IBM Z&reg; systems only)| - |
 | `SIGRTMAX -2 (62)`| Con  | Used by the `java.net` class library code                                         | - |
 | `SIGCHLD (17)`    | Con  | Used by the `java.lang.Process` implementation                     | - |
 
 :fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Notes:**
 
 - The use of `SIGRTMIN` is configurable with the `-Xdump:suspendwith=<num>` option.
 - The handling of `SIGABRT` is configurable with the `-XX[+|-]HandleSIGABRT` option.
+- While `SIGUSR2` signal is used for generating system dump files with exclusive access, which ensures reliable information for Java heap analysis, `SIGQUIT` signal is used for generating Java dump files with shared access for performance investigations. For more information, see [Dump events](xdump.md#dump-events).
```

Comment:
That might be a bit too strong. Users could use `SIGQUIT` for system dumps, and users could use `SIGUSR2` for Java dumps. Ultimately, both `SIGQUIT` and `SIGUSR2` can be customized however the user would like, so I think the goal here should just be to suggest a general best practice rather than concrete advice. That general best practice is that, given that `SIGQUIT` defaults to just a Java dump, and this is often useful for lightweight investigations, users now have the capability to use `SIGUSR2` for other heavy actions such as a System dump. Given that this page is just describing signals, I suggest removing this bullet.